tests/jsunit/xml_test.js

```diff
@@ -75,8 +77,35 @@ function test_domToWorkspace() {
         '  </block>' +
         '</xml>');
     var workspace = new Blockly.Workspace();
-    Blockly.Xml.domToWorkspace(dom, workspace);
+	Blockly.Xml.domToWorkspace(dom, workspace);
     assertEquals('Block count', 1, workspace.getAllBlocks().length);
+	Blockly.Xml.domToWorkspace(dom, workspace,[50,50]);
+	assertEquals('Block count', 2, workspace.getAllBlocks().length);
+  } finally {
+    delete Blockly.Blocks.test_block;
+  }
+}
+
+function test_appendDomToWorkspace() {
+  Blockly.Blocks.test_block = {
+    init: function() {
+      this.jsonInit({
+        message0: 'test',
+      });
+    }
+  };
+
+  try {
+    var dom = Blockly.Xml.textToDom(
+        '<xml xmlns="http://www.w3.org/1999/xhtml">' +
+        '  <block type="test_block" inline="true" x="21" y="23">' +
+        '  </block>' +
+        '</xml>');
+    var workspaceA = new Blockly.Workspace();
```

Comment:
I tried to use a WorkspaceSvg, instead of a Workspace for that commit but then I had trouble with getCanvas. I could not fine an example of the use of WorkspaceSvg in unit testing. Could you point one for me? 
Thank you for your help!